docs/tutorials/fairplay.md

```diff
@@ -76,3 +76,24 @@ player.getNetworkingEngine().registerResponseFilter((type, response) => {
   response.data = shaka.util.Uint8ArrayUtils.fromBase64(responseText).buffer;
 });
 ```
+
+## DRM Engine
+
+Some FairPlay support need to be detected in order to change source to HLS if FairPlay supported (AFAIK currently FairPlay is supported only within HLS and not publicly supported in DASH). You can query EME yourself, directly, and only query FairPlay, rather than the full list of key systems probed by `DrmEngine`. 
```

Comment:
nit: "need" -> "needs", "if FairPlay supported" -> "if FairPlay is supported".

Reply:
Thanks @michellezhuogg. Fixed it.